### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by dev-mamun**
- Use JavaScript classes.
- use medium-fidelity wireframes to create a UI.

### CODE REVIEW

**Review by brainconnect93 — COMMENTED** [review]
Hi @dev-mamun,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

- [ ] You have submitted a project that does not follow the pair programming requirement, kindly meet with your partner and let him/her also make commit in the project. Thank you

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

_Cheers and Happy coding!👏👏👏_

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

-----

_Invalid Code Review Request does not count into the code reviews limit._

File: js/books.js [hunk 1]
```diff
@@ -68,7 +60,7 @@ class Interface {
     const header = document.querySelector('.titletag');
     containerELement.insertBefore(newdiv, header);
     // Vanish in 3 seconds
-    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+    setTimeout(() => document.querySelector('.alert').remove(), 2000);
   }
 }
```
Comment (rloterh) [inline js/books.js:66]: - [ ] [OPTIONAL]
It's a good idea to use non mutating array methods `concat`, over `push`, and similarly `slice` over `splice` on _lines 82 and 91_ respectively, so as to keep the original array intact.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Daniel & Mamun,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Correctly refactor code to use classes!
- App runs correctly after code refactor!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- Kindly include details of the other co-contributor(s) in your ReadMe.
- Kindly endeavor to remove the guide note below the MIT license.

Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dev-mamun** [comment]
> * Kindly include details of the other co-contributor(s) in your ReadMe

@rloterh add as an author? right

**Pull request merged**